crm_claim_rma/migrations/9.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Shorten headers, like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py